AK/Atomic.h

```diff
@@ -31,6 +31,9 @@
 
 namespace AK {
 
+template <typename T> struct remove_volatile { typedef T type; };
+template <typename T> struct remove_volatile<volatile T> { typedef T type; };
```

Comment:
Please put these together with the other `RemoveFoo` helpers in `AK/StdLibExtras.h` :)

Reply:
Ah there they are. They're already there without me realizing it...